go.sum

```diff
@@ -100,6 +100,7 @@ github.com/PuerkitoBio/purell v1.1.1/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbt
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 h1:d+Bc7a5rLufV/sSk/8dngufqelfh6jnri85riMAaF/M=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
+github.com/Shopify/sarama v1.27.2 h1:1EyY1dsxNDUQEv0O/4TsjosHI2CgB1uo9H/v56xzTxc=
```

Comment:
Are they added automatically? Not sure why there were missing on main branch and suddenly added here.

Reply:
This test was failing on github 'Go / Check tidy, code generation and manifest (pull_request)'...so on running go mod tidy these lines were added....Should I remove them?

Reply:
Now the test is failing, I think they shouldn't be added.
https://github.com/antrea-io/antrea/actions/runs/3877253764/jobs/6612065974